cf-serverd/cf-serverd-functions.c

```diff
@@ -222,6 +222,7 @@ GenericAgentConfig *CheckOpts(int argc, char **argv)
                 GenericAgentWriteVersion(w);
                 FileWriterDetach(w);
             }
+// TODO re-use SafeExit() here for windows
```

Comment:
So why don't we do this here? Is there any reason why we shouldn't?

Reply:
I think we should. Just wasn't my focus during the bug fixing phase.